docs/release-procedure.md

```diff
@@ -59,14 +56,23 @@ Push the commit, make a PR _against `release-X.Y.Z`_ branch and, when it's merge
   * Push the tag to master: `git push upstream X.Y.Z`
     * If you're re-pushing a tag (beware! you shouldn't be doing that, no, _really_!), you can remove remote tag and push a local one by `git push upstream :refs/tags/X.Y.Z`
 
+***Note***: We are currently working on automating the release notes procedure - check 
+[#4747](https://github.com/modin-project/modin/issues/4747) for updates!
+
+### Update the link to release in readme
+
+**Note**: from now on you work in a branch in your fork (in `origin`).
+
+Change the `README.md` in the repo root so release badge points to new (would be created) release.
+
 ### Build wheel:
 
+**Note**: This should be done from your clean fork of Modin from the modin-project organization, where
+you made the release tag.
```

Comment:
that would make the changes without a PR, why is that?

Reply:
This is to build the package to upload to PyPi - @devin-petersohn prefers this to be done in a clean fork of modin-project/modin, and I think that makes sense!

Reply:
you should be saying "clean clone of `upstream`" or something similar, as "fork" is usually understood as a personal copy of some repo in Github, and this phrase should be about separate local clone

Reply:
Changed to clean clone!